padcli/command/deploy.go

```diff
@@ -162,6 +162,7 @@ func makeDeployOptions(
 			Values:        appValues,
 			Timeout:       lo.Ternary(len(jetCfg.Jobs()) > 0, 5*time.Minute, 0),
 		},
+		CreateNamespace:             !cluster.IsJetpackManaged(),
```

Comment:
oh aoh, we are introducing that method again. Maybe we can do `cluster.IsRemoteUnmanaged`? We don't need to create namespace for `docker-desktop` or do we